modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -356,5 +367,31 @@ class Instrument_List extends \NDB_Menu_Filter
             )
         );
     }
+
+    /**
+     * Generate a breadcrumb trail for that page.
+     *
+     * @return \LORIS\BreadcrumbTrail The trail.
+     */
+    public function getBreadcrumbs(): \LORIS\BreadcrumbTrail
+    {
+        $candid = $this->candidate->getCandID();
+        $pscid  = $this->candidate->getPSCID();
+
+        return new \LORIS\BreadcrumbTrail(
+            new \LORIS\Breadcrumb(
+                'Access Profile',
+                '/candidate_list'
+            ),
+            new \LORIS\Breadcrumb(
+                'Candidate Profile ' . $candid . ' / ' . $pscid,
```

Comment:
this would probably look cleaner if you used `"` and string interpolation instead of string literals and concats, but that's subjective and not really important..

Reply:
I agree. I will do it in a second pass